payjoin/src/receive/v1/mod.rs

```diff
@@ -740,10 +740,10 @@ impl ProvisionalProposal {
     pub fn finalize_proposal(
         mut self,
         wallet_process_psbt: impl Fn(&Psbt) -> Result<Psbt, Error>,
-        min_feerate_sat_per_vb: Option<FeeRate>,
-        max_feerate_sat_per_vb: FeeRate,
+        min_feerate: Option<FeeRate>,
+        max_effective_feerate: FeeRate,
```

Comment:
Should the `apply_fee` argument also be renamed to `max_effective_feerate`?

There are other places we're using `fee_rate` as an argument name instead of `feerate` and those should be made consistent.